Project-Omni-Ride-Network/Views/Home/Contact.cshtml

```diff
@@ -1,4 +1,37 @@
 @{
     ViewData["Title"] = "Kontakt";
 }
-<h1>Hello World</h1>
\ No newline at end of file
+
+<center>
+
+    <h1 class="home">Kontakt</h1>
+
+    <p>
+        Teilen Sie uns Ihr Anliegen mit. Wir helfen Ihnen gerne.
+    </p>
+
+</center>
+
+<img src="/images/train.png" id="zugzug" hidden="hidden" style="position: absolute; left: -4500px;"/>
+
+<script type="text/javascript" src="~/js/contact.js" ></script>
+<form action="@Url.Action("ContactAction", "Home")" method="POST" name="contactForm" id="contactForm">
+    
+    <center>
+        <input type="text" name="SenderName" id="SenderName" class="contact" placeholder="Max Mustermann" maxlength="100" required="required"/>
+
+        <input type="text" name="SenderEmail" id="SenderEmail" class="contact mailIn" placeholder="example@mail.de" maxlength="100"
```

Comment:
Still can hit send when email does not satisfy the regex.
Use 'pattern' attribute in the input tag (see register view as reference)